packages/experimental/src/components/AboutModal/AboutModal.mdx

```diff
@@ -19,43 +19,43 @@ completion.
 ### Default About screen modal
 
 <Preview>
-  <Story id="experimental-aboutmodal--default" />
+  <Story id="cloud-cognitive-canary-aboutmodal--default" />
 </Preview>
 
 ### About screen modal with links
 
 <Preview>
-  <Story id="experimental-aboutmodal--with-links" />
+  <Story id="xcloud-cognitive-canary-aboutmodal--with-links" />
```

Comment:
Are the `x`s in these id's a typo @dcwarwick?

Reply:
No, they are intentionally disabled because they break the story doc in its current form. The doc needs reworking, and we'll be opening an issue to tackle the stories and the story doc when we get to that.

Reply:
Got it 👍